src/core/tests/opset.cpp

```diff
@@ -161,6 +162,22 @@ TEST(opset, opset9_dump) {
     ASSERT_EQ(173, opset.get_types_info().size());
 }
 
+TEST(opset, opset10) {
+    auto op = std::make_shared<ov::opset10::Parameter>();
+    ASSERT_NE(nullptr, op);
+    EXPECT_TRUE(ov::op::util::is_parameter(op));
+}
+
+TEST(opset, opset10_dump) {
+    const auto& opset = ov::get_opset10();
+    std::cout << "All opset10 operations: ";
+    for (const auto& t : opset.get_types_info()) {
+        std::cout << t.name << " ";
+    }
+    std::cout << std::endl;
+    ASSERT_EQ(174, opset.get_types_info().size());
+}
```

Comment:
I think it is not needed. Please remove it

Reply:
Thanks, removed it. It was based on tests from opset9